docs/concepts/select.md

```diff
@@ -353,6 +353,61 @@ export class CityService {
 
 now we can use this `zooThemeParks` selector anywhere in our application.
 
+### Inheriting selectors  
+
+When we have states that share similar structure, we can extract the shared selectors into a base class which we can later extend from. If we have an `entities` field on multiple states, we can create a base class containing a dynamic `@Selector()` for that field, and extend from it on the `@State` classes like this.
+
+```TS
+export class EntitiesState {
+
+  static entities() {
```

Comment:
I may be wrong so please check, but I think that we could make this a getter and then we won't need to invoke the method to get the selector in the component... ?
```suggestion
  static get entities() {
```

Reply:
yup, i actually have it this way on my code :), updated